src/main/driver_unified.cpp

```diff
@@ -226,6 +227,8 @@ int runCvc5(int argc, char* argv[], std::unique_ptr<cvc5::Solver>& solver)
       {
         solver->setOption("wf-checking", "false");
       }
+      // now store options as original
+      pExecutor->storeOptionsAsOriginal();
```

Comment:
Should they be already stored from the previous call in line 190?

Reply:
This is two different branches of an `if-then-else` :)